test/alternator/README.md

```diff
@@ -76,3 +112,155 @@ cqlsh -x "INSERT INTO system_auth.roles (role, salted_hash) VALUES ('alternator'
+   It also makes the test code easier to read and understand.
+
+3. **Use fixtures to reduce test time**: When testing a feature with many
+   small test functions (as recommended above), often all of these small
+   test functions need some common setup, such as a table with a certain
+   schema or with certain data in it. Instead of each small test function
+   re-creating the same table (which takes time), use pytest _fixtures_.
+   Fixtures allow several test functions to use the same temporary test table.
+   Different tests can safely share a test table by using unique keys
+   instead of hard-coded keys that can break if another test accidentally
+   uses the same key. Because the DynamoDB data model is mostly schema-less,
+   we can often reuse the same table in many - even hundreds - of different
+   tests. test/alternator/conftest.py contains a few common fixtures like
+   that - for example `test_table_ss` is a table with a string hash key
+   and string sort key which many tests reuse.
+   The heavy use of fixtures - instead of creating a new table in each
+   tests - is one of the reasons why Alternator tests are so fast.
+   New tests should strive to keep it this way - and create as few as
+   possible new tables.
+
+4. **Write comments**: No, a self-explanatory test name is NOT enough.
+   It may be self-explanatory to you, but when the test fails a year
+   from now, nobody will remember what you were testing, or *why* you
+   decided to check these specific conditions, or if it's a forgotten
+   backport of a fix to an issue, which issue was that.
+   Before each test function, please explain why it exists - what feature
+   it intends to verify, and why it is tested in this specific way.
+   If the test is meant to reproduce a specific issue, please give the
+   issue number. All of this will be very helpful for later developers
+   which need to understand why your test suddenly failed after their
+   refactoring, or on a different Scylla branch.
+
+5. **Run your test against DynamoDB**: It is not enough to run your
+   test against Alternator and see that it passes. Run it against DynamoDB as
+   well, using `test/alternator/run --aws`. If the feature being tested is
+   Scylla-only, the test can be skipped on DynamoDB by using the `scylla_only`
+   fixture. But most of Scylla's Alternator features are identical to
+   DynamoDB's and therefore most of our Alternator tests should pass on
+   DynamoDB. If a test does not pass on DynamoDB, the test itself is likely
+   wrong, and should be fixed. In rare cases, the test fails on DynamoDB
+   because of a known bug in DynamoDB or a deliberate difference
+   between Scylla and DynamoDB; In these rare cases, the test can be
+   skipped by using the `scylla_only` fixture. However, make sure that
+   you explain in a comment why you did this.
+
+6. **Think about risky cases, don't just randomize**: One of the benefits
```

Comment:
```suggestion
6. **Think about edge cases, don't just randomize**: One of the benefits
```

Reply:
Well, I do use both words "risky" and "edge" several times in the following paragraph, I'm not sure which of the two words best summarizes the entire paragraph, I thought that "risky" was good but word choice isn't a precise science :-)